frontend/src/app/components/workflow-editor/workflow-editor.component.ts

```diff
@@ -261,31 +259,65 @@ export class WorkflowEditorComponent {
     if (element.getAttribute('dispatcher') !== 'link') {
       return;
     }
-    const link = this.currentWorkflow.links.find(
+    const link = this.currentWorkflow.content.links.find(
       wfLink => wfLink.id === element.id
     );
     if (link !== undefined) {
       return;
     }
-    const linkSourceIds = this.flowchartConverter.getLinkOperatorAndConnector(
+
+    const linkSourceIds: VertexIds = this.flowchartConverter.getLinkOperatorAndConnectorId(
       element,
       true
     );
-    const linkTargetIds = this.flowchartConverter.getLinkOperatorAndConnector(
+    const linkTargetIds: VertexIds = this.flowchartConverter.getLinkOperatorAndConnectorId(
```

Comment:
Link ziehen von WorkflowOutput zu Operator-Output, dann sichergehen, dass der WorkflowOutput hier in den TargetIds steht.